### PR TITLE
Update notebooks

### LINKED ISSUE #91: get_available_templates is not defined
`OrionDBExplorer Tutorial.ipynb`:
image

Error message:
`ImportError: cannot import name 'get_available_templates'`

### LINKED ISSUE #176: Update notebooks
Since their first initiation, the code behind the underlying notebooks are outdated. 

Create a PR to update
* how pipelines get loaded.
* remove deprecated functions.
* ensure they run end-to-end without error.

### PR DESCRIPTION
**Pull request by sarahmish**
Resolve #91.
Resolve #176.

### CODE REVIEW

File: notebooks/OrionDBExplorer Tutorial.ipynb [hunk 1]
```diff
@@ -462,9 +455,9 @@
     }
    ],
    "source": [
-    "from orion import get_available_templates\n",
+    "from orion.analysis import get_available_templates\n",
     "\n",
-    "get_available_templates()"
+    "get_available_templates(None)"
```
Comment (dyuliu) [inline notebooks/OrionDBExplorer Tutorial.ipynb:460]: Why we need to pass None here?

File: notebooks/Pipeline Scoring Example.ipynb [hunk 2]
```diff
@@ -363,49 +311,50 @@
    "metadata": {},
    "source": [
     "Afterwards, we pass the ground truth, the detected anomalies and the original data\n",
-    "to the `orion.metrics.accuracy_score` and `orion.metrics.f1_score` functions in order\n",
+    "to the `orion.metrics.contextual_accuracy` and `orion.metrics.contextual_f1_score` functions in order\n",
```
Comment (dyuliu) [inline notebooks/Pipeline Scoring Example.ipynb:314]: Do we have a document introducing the definitions of the metrics? If yes, we can provide the links here.

**Review by dyuliu — APPROVED** [review]

File: notebooks/OrionDBExplorer Tutorial.ipynb (hunk 1 above)
Comment (sarahmish) [reply notebooks/OrionDBExplorer Tutorial.ipynb]: That was a test from my end, I removed it

File: notebooks/Pipeline Scoring Example.ipynb (hunk 2 above)
Comment (sarahmish) [reply notebooks/Pipeline Scoring Example.ipynb:314]: Done!

**Pull request merged**